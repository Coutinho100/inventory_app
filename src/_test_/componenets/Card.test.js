// import React from 'react';
// import Card from '../../components/Component/Card';
// import { View } from 'react-native';
// import { configure, mount, shallow } from 'enzyme';
// import Adapter from 'enzyme-adapter-react-16';
// import renderer from 'react-test-renderer';
// configure({ adapter: new Adapter() });


// describe("<card view/>", () => {


//     test('renders correctly', () => {
//         const tree = renderer.create(<Card />).toJSON();
//         expect(tree).toMatchSnapshot();
//     });
//     it('renders the length of view', () => {
//         const wrapper = shallow(<Card />);
//         expect(wrapper.find(View).length).toEqual(1);
//     })

// });


import React from 'react';
import Card from '../../components/Component/Card';
import { configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { View} from 'react-native';
import test_function from './test_function';

configure({ adapter: new Adapter() });
test_function(["<card view/>",1],['renders correctly','renders the length of view'],[<Card />,View],null)





