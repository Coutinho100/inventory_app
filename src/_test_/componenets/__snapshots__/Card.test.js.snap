// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<card view/> renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "borderRadius": 10,
      "elevation": 5,
      "shadowColor": "black",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.26,
      "shadowRadius": 8,
    }
  }
/>
`;
